docs/spec-files/appendix.md

```diff
@@ -0,0 +1,671 @@
+## Appendix
```

Comment:
Appendix's traditionally are used to add additional information and are therefore considered non-normative. Additionally, the purpose of normative statements have most often been used to call out what's testable so making normative statements about how to test something is a bit recursive.

We should remove any normative statements in this portion.